aws/build_and_mail_log

```diff
@@ -35,7 +35,7 @@ def main()
     commit_url = "https://github.com/code-dot-org/code-dot-org/commit/#{RakeUtils.git_revision}"
     # Notify the HipChat channels about what happened.
     projects = ARGV.join(' ')
-    time_message = " &#x1f550; #{format_duration(Time.now.to_i - start_time)}"
+    time_message = " 🕐 #{format_duration(Time.now.to_i - start_time)}"
```

Comment:
Did you really want this change?  It seems unrelated to other change and not sure we need it.  Might also lead to issues if some tool gets confused about encoding of this file.